### PR TITLE
Add aarch64 wheel build support

### LINKED ISSUE #91: Wheel support for linux aarch64 [arm64]
Summary
Installing py-setproctitle on aarch64 via pip using command "pip3 install py-setproctitle" tries to build wheel from source code

Problem description
py-setproctitle doesn't have wheel for aarch64 on PyPI repository. So, while installing py-setproctitle via pip on aarch64, pip builds wheel for same resulting in it takes more time to install py-setproctitle. Making wheel available for aarch64 will benefit aarch64 users by minimizing py-setproctitle installation time.

Expected Output
Pip should be able to download py-setproctitle wheel from PyPI repository rather than building it from source code.

@py-setproctitle-team, please let me know if I can help you building wheel/uploading to PyPI repository. I am curious to make py-setproctitle wheel available for aarch64. It will be a great opportunity for me to work with you.

### PR DESCRIPTION
**Pull request by odidev**
Add aarch64 wheel build support in github action using qemu (closes #91 )

### CODE REVIEW

File: .github/workflows/packages.yml [hunk 1]
```diff
@@ -12,13 +12,25 @@ jobs:
       - name: Build sdist
         run: python setup.py sdist
 
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+
+      - name: Available platforms
+        run: echo ${{ steps.qemu.outputs.platforms }}
```
Comment (dvarrazzo) [inline .github/workflows/packages.yml:20]: I assume this is a debug statement?

**Comment by dvarrazzo** [comment]
I have separated every platform's build into a different job, so they can run in parallel and requirements can be different.

I would be interested in testing the packages built. I have opened a question here: https://github.com/RalfG/python-wheels-manylinux-build/issues/34

Testing the artifacts is not something strictly related to the arm build, but the more exotic the platform, the more I'm nervous about releasing something broken and I would have some guarantee that it works as expected :D

Have you got ideas about how we could run these tests piggybacking these build images?

File: .github/workflows/packages.yml (hunk 1 above)
Comment (odidev) [reply .github/workflows/packages.yml]: docker/setup-qemu-action@v1 is required to run aarch64 docker container on x86 builders. However we can remove line 19 and 20 which is just listing what platforms qemu supports.

**Comment by dvarrazzo** [comment]
Hi @odidev 

Windows people are expecting a release, and their packages have been contributed for quite a while, so I would like to release them soon.

I will release the arm packages too, however I consider them untested. If arm people find problem I will ask you to look into it :smile:

**Comment by odidev** [comment]
@dvarrazzo thanks for considering to release arm64 wheels. I have executed the tests on arm64 machine and all the test cases are passing successfully. Please have a look at attached logs: [arm64-test-logs.txt](https://github.com/dvarrazzo/py-setproctitle/files/5856439/arm64-test-logs.txt)

**Pull request merged**